starters/deno-oak-denodb/src/graphql/resolvers/mutation_handler.ts

```diff
@@ -15,17 +18,26 @@ export const createTechnology = async (
 export const updateTechnology = async (
 	_: unknown,
 	{ id, input }: { id: string; input: TechnologyInput },
-) => {
+	{ ds }: { ds: Cache },
+	info: Record<string, string>,
+): Promise<{ done: boolean }> => {
 	await Technologies.where('id', id).update({
 		...input,
 	});
+	await ds.invalidateItem(`${info.fieldName}:${id}`);
```

Comment:
When updating an entry, should you invalidate the `getTechnologies` entry as well? 🤔